client/state/reader/posts/normalization-rules.js

```diff
@@ -107,6 +107,30 @@ export function classifyPost( post ) {
 	return post;
 }
 
+/**
+ * Add discover properties to a post
+ * @param  {Object} post - the post to extend
+ * @return {Object}      - the post with discover properties
+ */
+
+export function addDiscoverProperties( post ) {
+	const isDiscover = !! ( get( post, 'discover_metadata' ) || DISCOVER_BLOG_ID === get( post, 'site_ID' ) );
```

Comment:
Yeah it does work the same but if we were to choose one place do this check I would suggest we do it in the post normalizer.  My thought was to remove the helper version after we start reading `isDiscover` directly from the post object.

Also want to consolidate the discover blog id which is defined here in the normalizer and a few other places